src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -80,7 +81,7 @@ def process_event(project_id: int, group_id: int, event_id: str) -> list[CodeMap
         trees = get_trees_for_org(installation, org, extra)
         trees_helper = CodeMappingTreesHelper(trees)
         code_mappings = trees_helper.generate_code_mappings(frames_to_process, platform)
-        create_repos_and_code_mappings(code_mappings, installation, project, platform)
```

Comment:
We're also going to be creating in_app stack trace rules, thus, renaming in advance.